closed/autoconf/custom-hook.m4

```diff
@@ -308,9 +308,8 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
 AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
   OPENJDK_SHA=`git -C $TOPDIR rev-parse --short HEAD`
-
-  # Find first rev tagged by jdk-12* but not containing "_openj9"
-  LAST_TAGGED_SHA=`git -C $TOPDIR rev-list --exclude="*_openj9*" --tags="jdk-${VERSION_FEATURE}*" --topo-order --max-count=1 2>/dev/null`
+  # Find first rev tagged by jdk-11* but not containing "_openj9"
+  LAST_TAGGED_SHA=`git -C $TOPDIR rev-list --exclude="*_openj9*" --tags="jdk-11*" --topo-order --max-count=1 2>/dev/null`
```

Comment:
This should *not* change: we want the tag for jdk12.

Reply:
Whoops, agree this needs to be fixed. Will do that right now.

Reply:
Done